README.md

```diff
@@ -35,65 +61,23 @@ registerForVideoEffect(callback)
   ) 
   ```
   - Whenever the user selects a different effect in a video app, the video app should call this API to notify Teams client. 
-## How to use the test app?
-
-
-1. Test-app was used to develop/verify video app, you can find the test-app under the `test-app` folder, copy and unzip to other folder. Ensure you are in Windows, and your computer has a camera. Use OBS virtual camera if you don't have a camera.
-1. After the application is opened, select a camera device in the 'Camera' drop down.
-1. Input your video app's URL. Deploying the video app in https server is preferred.
-1. Click `Load video app`. The sample video app will be loaded if the `Video app url` is blank.
 
-## How to test performance?
+## How to test performance in test app?
 
-1. Load video app following commands in the above section.
-2. To evaluate processing time, click `Real-time Evaluation` or `Full Evaluation` under `Time per Frame'. 
+1. Load video app in test app.
+2. To evaluate processing time, click `Real-time Evaluation` or `Full Evaluation` under `Time per Frame`. 
     - `Real-time Evaluation` logs the average and range of processing time in millisecond in each second.
     - `Full Evaluation` logs both the processing time and the distribution of processing time.
 3. To evaluate memory usage, click `Real-time Evaluation` or `Full Evaluation` under `Memory Usage'. 
     - `Real-time Evaluation` logs the average and range of both active heap size and total heap size for each frame.
     - `Full Evaluation` logs both the heap size and the distribution of heap size.
 4. To test video app under different resolutions, click `Reduce the resolution by half`, `Same`, or `Double the resolution`. 
 
-
-## How to change to your own video app?
-
-1. Click `Destroy video app` if you have already loaded the sample video app.
-2. Input your video app's URL.
-3. Click `Load video app`.
-
-## How to change camera resolution?
-
-We can change the video frame resolution by changing this configuration file: `%appdata%\Roaming\Microsoft\electron\SkypeRT\persistent.conf`
-
-Please append the following key-value to the json file: `"RtmCodecsConfig": {"MinVideoSourceResolution":"83886800", "MaxVideoSourceResolution":"83886800"}`
-
-Here are the allowed number corresponding to specific resolution:
-
-```
-(640, 360)   = 0x02800168 = 41943400
-(1280, 720)  = 0x050002D0 = 83886800
-(1920, 1080) = 0x07800438 = 125830200
-```
-
-After saving the persistent.conf, restart the test-app, it should grab video frame from camera with the given resolution.
- 
-The RtmCodecsConfig key-value will be removed every time restarting the test-app, it's expected. But the resolution will be remembered in the test-app future running.
-
-## There's no device shown in the 'Camera' dropdown.
+## FAQ: There's no device shown in the 'Camera' dropdown.
 
 1. Make sure your camera has been plugged into your computer.
 2. Delete `%appdata%\Microsoft\electron` folder.
 
-## Install the video app in Teams
```

Comment:
Should we remove the sideloading instruction section?

Reply:
We have many other documentations to teach user how to sideload to Teams client. I think this cannot be covered in sample app repo.